pkg/tools/gen/genkcl_jsonschema.go

```diff
@@ -419,7 +430,15 @@ func convertSchemaFromJsonSchema(ctx *convertContext, s *jsonschema.Schema, name
 	if result.HasIndexSignature && result.IndexSignature.validation != nil {
 		result.Validations = append(result.Validations, *result.IndexSignature.validation)
 	}
-	result.property.Name = strcase.ToSnake(result.Name)
+	switch ctx.casingOption {
```

Comment:
The CI faild. Could you please fix it and add more unit tests for different options? Thank you!